### ISSUE TITLE
Summarize what is needed for PEcAn VICE

### ISSUE THREAD

**Issue by dlebauer** [az-digitalag/organization#123]
Gather requirements: what new features might be required in CyVerse?

* Need PEcAn Docker container deployed on VICE
* Container needs access to
   * ED2 and BioCro models: Should we run these in Docker locally or on welsch or on UA campus cluster? Will frequently require 500 runs, up to 1m runs for regional simulations
   * Access to either BETYdb on Welsch or a local copy (this is on Docker, public users should have their own copy of the database)

PEcAn has a framework for remote execution: 
* Overview https://pecanproject.github.io/pecan-documentation/develop/pecan-remote.html 
* XML settings https://pecanproject.github.io/pecan-documentation/develop/pecanXML.html#xml-host
* "Model-launcher" for executing lots of runs in one submission https://github.com/PecanProject/pecan/tree/develop/contrib/modellauncher

**Comment by KristinaRiemer** [comment]
See [additional HackMD notes](https://hackmd.io/ms-iQBtnQD-MfbMdO6TbjA?view)

**Comment by julianpistorius** [comment]
@dlebauer can you please review this when you have some time?

**Comment by dlebauer** [comment]
Definitely ready to get started. A few thoughts:

1. It isn't clear what problem using pglite solves relative to our current dockerized setup (allowing each user to have their own database? I am not sure what is involved in using pglite, but it isn't clear what the advantage is. I think better to have options to use either a sandbox or a production instance (like welsch) would be sufficient. 
2. CA should be a separate epic, and we should engage the rest of the team in this (e.g. first step is to get into VICE, and then present what we have and the CA proposal to the PEcAn call).

**Comment by julianpistorius** [comment]
Hi @dlebauer 

1. Yup. You're right about `pglite`. It would give every user their own database. The idea being that they can start from a clean slate. Especially useful when somebody like Tonantzin wants to reproduce Kristina's work. A dirty database state could cause problems otherwise. Using `pglite` is relatively straightforward. Another alternative is using the Nomad jobs I made so each person can easily create their own persistent PostgreSQL & Bety setup (the sandbox you mentioned) to which they can then connect from VICE.
2. Yes, CA should be a separate epic. The reason we bundled them together in the HackMD was to identify how much overlap there is - which is a lot! But getting VICE done first is the right thing to do, and now we know it will also get us most of the way to getting CA done once we start that. It will also help to keep the CA use case in mind when building the VICE app so when we are faced with making implementation choices we can think about which otherwise equivalent choices would make the CA use case easier or harder.

Hope that makes sense. Thanks for the feedback! That was useful.
